lucid/misc/io/reading.py

```diff
@@ -199,11 +199,11 @@ def _read_and_cache(url, mode="rb"):
 
 
 from functools import partial
-from io import DEFAULT_BUFFER_SIZE
+_READ_BUFFER_SIZE = 1048576
```

Comment:
Can we just add a short comment why this number and not the default so I remember in the future?